subsys/net/lib/nrf_cloud/src/nrf_cloud_pgps.c

```diff
@@ -371,9 +371,15 @@ static void discard_oldest_predictions(int num)
 	npgps_print_blocks();
 
 	/* update index and header for new first stored prediction */
+	uint16_t gps_day;
+	uint32_t gps_time_of_day;
+
 	get_prediction_day_time(last, &index.start_sec,
-				&index.header.gps_day,
-				&index.header.gps_time_of_day);
+				&gps_day,
+				&gps_time_of_day);
+	index.header.gps_day = gps_day;
```

Comment:
could you add a comment here to indicate why the intermediate variables are used?